include/deal.II/lac/solver_cg.h

```diff
@@ -166,6 +182,33 @@ class SolverCG : public Solver<VECTOR>
          const VECTOR         &b,
          const PRECONDITIONER &precondition);
 
+  /**
+   * Connect a slot to retrieve the CG coefficients. The slot will be called
+   * with alpha as the first argument and with beta as the second argument.
+   * Called once per iteration.
+   */
+  boost::signals2::connection
+  connect_coefficients_slot(
+    const std_cxx11::function<void (double,double)> &slot);
+
+  /**
+   * Connect a slot to retrieve the estimated condition number.
+   * Called on each iteration if every_iteration=true otherwise called
+   * after all iterations.
```

Comment:
"after all iterations" is misunderstandable. How about "otherwise called at the end of iterations once convergence has been achieved"?